terraform/aws/addons/templates/keycloak-operator.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: v1
```

[p5] Good to have separator on top of file for hypothetical case when YAML's will be merged.
```suggestion
---
apiVersion: v1
```